src/index.spec.ts

```diff
@@ -327,6 +327,52 @@ describe("ServerlessPlugin", () => {
     });
   });
 
+  it("Adds tracing layer for dotnet", async () => {
+    mock({});
+    const serverless = {
+      cli: {
+        log: () => {},
+      },
+      service: {
+        provider: {
+          region: "us-east-1",
+        },
+        functions: {
+          node1: {
+            handler: "my-func.ev",
+            layers: [],
+            runtime: "dotnet6",
+          },
+        },
+        custom: {
+          datadog: {
+            apiKey: 1234,
+          },
+        },
+      },
+    };
+
+    const plugin = new ServerlessPlugin(serverless, {});
+    await plugin.hooks["after:package:initialize"]();
+    expect(serverless).toMatchObject({
+      service: {
+        functions: {
+          node1: {
+            handler: "my-func.ev",
+            layers: [
+              "arn:aws:lambda:us-east-1:464622532012:layer:Datadog-Extension:21",
+              "arn:aws:lambda:us-east-1:464622532012:layer:dd-trace-dotnet:1",
```

Comment:
I think this test might fail when we update the extension version (or dd-trace-dotnet version). Do we do something in other tests to handle this?